webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala

```diff
@@ -1425,8 +1425,8 @@ class ProjectsResponderADM(responderData: ResponderData) extends Responder(respo
    * @param project a [[ProjectADM]].
    * @return true if writing was successful.
    */
-  private def writeProjectADMToCache(project: ProjectADM): Future[Boolean] = {
-    val result = (storeManager ? CacheServicePutProjectADM(project)).mapTo[Boolean]
+  private def writeProjectADMToCache(project: ProjectADM): Future[Unit] = {
+    val result = (storeManager ? CacheServicePutProjectADM(project)).mapTo[Unit]
```

Comment:
Not really a useful information. If it succeeds then it will return a Unit and if it fails, then there will be an exception in the error channel. Before it returned either `true` or an exception.